src/modules/market/components/Products/useProductsStyles.ts

```diff
@@ -0,0 +1,50 @@
+import { makeStyles } from "@material-ui/core";
+
+interface StyleProps {
+  firstTitle: string;
+  lastTitle: string;
+}
+
+export const useProductsStyles = makeStyles((theme) => ({
+  root: (props: StyleProps) => ({
+    marginTop: '44px',
+    display: 'flex',
+    justifyContent: 'flex-end',
+
+    '& .MuiPaginationItem-root': {
```

Comment:
Try to use [PaginationItem classes api](https://material-ui.com/api/pagination-item/#css) for style definitions instead of css cascade.